docs/options.md

```diff
@@ -416,43 +416,26 @@ Default: reads your package's Python compatibility from `pyproject.toml`
 `setup.py` `setup(python_requires="...")`. If not found, cibuildwheel assumes
 the package is compatible with all versions of Python that it can build.
 
-
 !!! note
-    Rather than using this option, it's recommended you set
-    `project.requires-python` in `pyproject.toml` instead:
-
-    ```toml
-    [project]
-    requires-python = ">=3.6"
-    ```
+    Rather than using this option, it's recommended you set this value
+    statically in a way that your build backend can use it, too. This ensures
+    that your package's metadata is correct when published on PyPI.
```

Comment:
```suggestion
    Rather than using this option, it's recommended you set this value
    statically in a way that your build backend can use it, too. This ensures
    that your package's metadata is correct when published on PyPI. This option
    is provided by cibuildwheel as an emergency override, and therefore is only
    available in environment variable form.
```

Reply:
```suggestion
    Rather than using this environment variable, it's recommended you set this value
    statically in a way that your build backend can use it, too. This ensures
    that your package's metadata is correct when published on PyPI. This override
    is provided by cibuildwheel only for emergencies, and therefore is only
    available in environment variable form.
```

I suggest a modest rewrite of the above suggestion to be more clear, avoid repetition and read more cleanly. (In particular, it confused me as to what exactly "this option" was referring to, since it was several paragraphs distant from the original reference, and the intervening ones had talked about the static methods of setting this value)

Reply:
Thanks both. I'll take @CAM-Gerlach's version, but just tweak 'emergencies', as it seems a bit strong. 

```suggestion
    Rather than using this environment variable, it's recommended you set this value
    statically in a way that your build backend can use it, too. This ensures
    that your package's metadata is correct when published on PyPI. This 
    cibuildwheel-specific option is provided as an override, and therefore is only
    available in environment variable form.
```

Reply:
Sounds good; I wasn't too fond of that wording either to be honest.